client/src/components/Register.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  input {
+    width: 60%;
+    padding: 15px;
+    margin: 5px 0;
+    border-radius: 20px;
+    background-color: ${props => props.theme.lBlue}
+  }
+
+`;
+
+
+class RegisterLogin extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      user: {
+        username: '',
+        password: '',
+      },
+      showModal: false,
+      modalMessage: '',
+    }
+  }
+
+  clearFormFields = () => {
+    this.setState({ user: {username: '', password: ''} })
+  }
+
+  handleChange = (e) => {
+    const value = {[e.target.name]: e.target.value};
+    this.setState({ user: {...this.state.user, ...value}})
+  }
+
+  handleSubmit = (e) => {
+    this.setState({ modalMessage: '' })
+    e.preventDefault();
+    if (this.props.login) {
+      axios.post('http://localhost:5000/api/login', this.state.user)
+        .then(res => {
+          this.setState({modalMessage: res.data, showModal: true})
```

Comment:
It looks like you're pretty far on your front-end, you should show this in stand-up. (I didn't realize you had a pop-up modal working, stuff like that really adds to the overall feel of the site)